tools/nodejs_api/sample.js

```diff
@@ -0,0 +1,14 @@
+// Make sure the test directory is removed as it will be recreated
+const fs = require("fs");
+const {Database, Connection} = require("./src_nodejs/index.js");
```

Comment:
You should probably use CMake to copy the built module and the js files into a proper directory structure (refer to https://github.com/kuzudb/kuzu/blob/89fba7c1328b543747fc4948e6bff0961263276b/tools/python_api/CMakeLists.txt#L32) so you can do something like `require("build/kuzu/")`

Reply:
Is it ok if I do require("build/kuzu/index.js")?

Reply:
nvm